### ISSUE TITLE
ManimPango could not import and load the necessary shared libraries.

### ISSUE THREAD

**Issue by frederik-markus** [esling/creative_ml#1]
Hello,

I am having some trouble getting the environment  set up. After setting up my environment using:

`conda create -n <environment_name> python=3.9`

I then run: 

`pip install -r requirements.txt`

during which I get the following error :

 The 'sklearn' PyPI package is deprecated, use 'scikit-learn'
      rather than 'sklearn' for pip commands.

In an attempt to amend this I therefore changed the package name from sklearn to scikit-learn in requirements.txt. After changing this I can successfully run the pip install command as well and subsequently pip install jupyterlab as well. I then try to verify the correctness of the setup by running each box in the setup notebook which works up to the "Interactive Course Aspects" section, where I receive the following error: 

```
ImportError                               Traceback (most recent call last)
File ~/anaconda3/envs/creative_ml/lib/python3.9/site-packages/manimpango/__init__.py:14
     13 try:
---> 14     from .cmanimpango import *  # noqa: F403,F401
     15     from .enums import *  # noqa: F403,F401

ImportError: /lib/x86_64-linux-gnu/libgobject-2.0.so.0: undefined symbol: ffi_type_uint32, version LIBFFI_BASE_7.0

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
Cell In[7], line 2
      1 import holoviews
----> 2 import manim
      3 import panel
      4 import rise

File ~/anaconda3/envs/creative_ml/lib/python3.9/site-packages/manim/__init__.py:32
     30 from .animation.indication import *
     31 from .animation.movement import *
---> 32 from .animation.numbers import *
     33 from .animation.rotation import *
     34 from .animation.specialized import *

File ~/anaconda3/envs/creative_ml/lib/python3.9/site-packages/manim/animation/numbers.py:10
      5 __all__ = ["ChangingDecimal", "ChangeDecimalToValue"]
      8 import typing
---> 10 from manim.mobject.text.numbers import DecimalNumber
     12 from ..animation.animation import Animation
     13 from ..utils.bezier import interpolate

File ~/anaconda3/envs/creative_ml/lib/python3.9/site-packages/manim/mobject/text/numbers.py:15
     13 from manim.mobject.opengl.opengl_compatibility import ConvertToOpenGL
     14 from manim.mobject.text.tex_mobject import MathTex, SingleStringMathTex, Tex
---> 15 from manim.mobject.text.text_mobject import Text
     16 from manim.mobject.types.vectorized_mobject import VMobject
     17 from manim.mobject.value_tracker import ValueTracker

File ~/anaconda3/envs/creative_ml/lib/python3.9/site-packages/manim/mobject/text/text_mobject.py:64
     61 from pathlib import Path
     62 from typing import Iterable, Sequence
---> 64 import manimpango
     65 import numpy as np
     66 from colour import Color

File ~/anaconda3/envs/creative_ml/lib/python3.9/site-packages/manimpango/__init__.py:35
     18     py_ver = ".".join(map(str, sys.version_info[:3]))
     19     msg = f"""
     20 
     21 ManimPango could not import and load the necessary shared libraries.
   (...)
     33 Original error: {ie}
     34 """
---> 35     raise ImportError(msg)

ImportError: 

ManimPango could not import and load the necessary shared libraries.
This error may occur when ManimPango and its dependencies are improperly set up.
Please make sure the following versions are what you expect:

    * ManimPango v0.4.3, Python v3.9.16

If you believe there is a greater problem,
feel free to contact us or create an issue on GitHub:

    * Discord: https://www.manim.community/discord/
    * GitHub: https://github.com/ManimCommunity/ManimPango/issues

Original error: /lib/x86_64-linux-gnu/libgobject-2.0.so.0: undefined symbol: ffi_type_uint32, version LIBFFI_BASE_7.0
```
I believe that this is an issue with the manimpango package, but I have not been able to fix this.

Please advise on how to proceed.

Thanks.

**Comment by acids-ircam** [comment]
Thanks for the catch on scikit-learn, I updated the requirements file.

Regarding `Manim`, it is an optional library for the course as it is the one that I use to generate the animation that you can find in the slides, so we will not use it for the course exercises at all.

That being said, on discord, @Matteo Fantozzi seemed to have a similar issue for Ubuntu 22.04 and solved it the following way:
The problem is that pip fails to install `manim` because of one of its dependencies, `manimpango`.
I found that manually building and installing `manimpango` will solve this. 
I followed the building steps here: [https://github.com/ManimCommunity/ManimPango#building](https://github.com/ManimCommunity/ManimPango#building)

In short, I only needed two commands:

`sudo apt install libpango1.0-dev pkg-config python3-dev`

`pip install manimpango --no-binary :all:`

Please confirm if that solves your problem so we can close the issue ;)

**Comment by frederik-markus** [comment]
So I have managed to fix the manim issue using the steps suggested above. The issue can be closed.
